.env

```diff
@@ -1,3 +1,3 @@
 NEXT_PUBLIC_AUTH0_DOMAIN=maffin-dev.eu.auth0.com
 NEXT_PUBLIC_AUTH0_CLIENT_ID=mMmnR4NbQOnim9B8QZfe9wfFuaKb8rwW
-NEXT_PUBLIC_AUTH0_SCOPES=profile email
+NEXT_PUBLIC_AUTH0_SCOPES=profile email https://www.googleapis.com/auth/drive.file
```

Comment:
regardless of the env now users can be premium so google drive permission is needed